register-api/src/main.py

```diff
@@ -79,6 +79,7 @@ async def register_user(item:Item):
         if checkerica['exist'] == False:
             CreateUser(item.email, item.username, item.firstName, item.lastName, item.secret).new_user()
             check_user_id = CreateUser(item.email, item.username, item.firstName, item.lastName, item.secret).get_keycloak_user_id()
+            CreateUser(item.email, item.username, item.firstName, item.lastName, item.secret).create_keycloak_roles()
```

Comment:
Ako ovo ovako ostavis, to znaci da ce prilikom svakog kreiranja user-a da se opali komanda da kreira user_role, sto zapravo nema nikakvog smisla.
Tako da, na ovom mestu ne treba da se poziva definicija CreateUser nego AssingRole, tj dodeljivanje vec postojece role.